sys-auth/AusweisApp2/AusweisApp2-1.20.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Official authentication app for German ID cards and residence permits"
+HOMEPAGE="https://www.ausweisapp.bund.de/"
+SRC_URI="https://github.com/Governikus/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EUPL-1.2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
```

Comment:
You are missing dependencies. It's also linked against udev. Since it's not part of base profile, it needs to be added. See ```virtual/udev/udev```
```  NEEDED               libudev.so.1```

Reply:
As dev-qt/qtdeclarative is a dependency of dev-qt/qtquickcontrols2, do we still have listed in RDEPEND?

Reply:
yes.

Reply:
ok, I will add it